package.json

```diff
@@ -90,8 +90,8 @@
                     "source.tsx",
                     "text.html.basic"
```

Comment:
I'm not too familiar with how this works, but should this `injectTo` field maybe be changed to `meta.embedded.block.html` to match the `injectionSelector`?

Reply:
I don't think so. `injectTo` specifies that target grammar id while the injectionSelector specifies the scope within the grammar to inject into